source/MaterialXTest/GenGlsl.h

```diff
@@ -0,0 +1,55 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#include <MaterialXTest/Catch/catch.hpp>
+
+#include <MaterialXGenGlsl/GlslShaderGenerator.h>
+#include <MaterialXGenGlsl/GlslSyntax.h>
+
+#include <MaterialXTest/GenShaderUtil.h>
+
+namespace mx = MaterialX;
+
+// Base GLSL code generation tester
+class GLSLGenCodeGenerationTester : public GenShaderUtil::ShaderGeneratorTester
```

Comment:
Nitpick: For consistency with the base class naming, and shader generator class naming, maybe this class could be named `GlslShaderGeneratorTester`